library/include/G4CMPUtils.hh

```diff
@@ -65,7 +65,7 @@ namespace G4CMP {
   G4bool IsHole(const G4ParticleDefinition& pd);
   G4bool IsChargeCarrier(const G4ParticleDefinition& pd);
 
-  // Select phonon mode randomly from density of states
+  // Select phonon index randomly from density of states
```

Comment:
Can you put this back?  It's just the argument to `LambertianReflection()` that needed to change (below).